### PR TITLE
Portuguese translation of base template

### PR DESCRIPTION
**Pull request by HakaCode**
I hope I did everything correctly.

Makes progress on #505

### CODE REVIEW

File: src/templates/pt/2019/base_chapter.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+{% block prev_next_title %}Navegação do capítulo anterior e seguinte{% endblock %}
+
+{% block discuss_this_chapter %}Discuta este capítulo{% endblock %}
+{% block comment %}Comente{% endblock %}
+{% block comments %}comentários{% endblock %}
+{% block queries %}Ver consultas{% endblock %}
+{% block results %}Ver resultados{% endblock %}
+
+{% block previous %}Capítulo anterior (pressione 'p' ou ','){% endblock %}
+{% block next %}Próximo Capítulo (pressione 'n' ou '.'){% endblock %}
```
Comment (bazzadp) [inline src/templates/pt/2019/base_chapter.html:20]: That telltale red mark shows we need a new line at the end of this file (some programs complain about missing new lines hence best practice to always end a text file with one - which is why GitHub highlights it when you don't).

```suggestion
{% block next %}Próximo Capítulo (pressione 'n' ou '.'){% endblock %}

```

**Comment by HakaCode** [comment]
> Just one small correction please!
> 
> BTW, see my [Git Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Git-Guide#recommended-way-to-use-git-for-the-web-almanac) for ways to avoid that long commit trail

Yes, i spent hours trying to learn by reading and putting it into practice even though i have never seen it, sometimes i get lost with some commands, however, i haven’t done so far, i use Google Translate :)

**Comment by HakaCode** [comment]
@bazzadp  could you guide me on this?

```bash
$ git fetch upstream
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by bazzadp** [comment]
This suggests you haven’t set up your ssh key. We’re you using https access to the repo (did you have to type password each time you wanted to push changes?)?

If so, then you needed to add `upstream` in the same way using https instead of git.

You can switch this to https, using:

`git remote set-url upstream   https://github.com/HTTPArchive/almanac.httparchive.org.git`

It us worth looking into setting up SSH keys though and using `git@github.com` addresses instead of `https://github.com`. Much better as don’t have to type your password each time.

**Comment by bazzadp** [comment]
This suggests you haven’t set up your ssh key. We’re you using https access to the repo (did you have to type password each time you wanted to push changes?)?

If so, then you needed to add `upstream` in the same way using https instead of git.

You can switch this to https, using:

`git remote set-url upstream   https://github.com/HTTPArchive/almanac.httparchive.org.git`

It us worth looking into setting up SSH keys though and using `git@github.com` addresses instead of `https://github.com`. Much better as don’t have to type your password each time.

**Comment by HakaCode** [comment]
I got it, thank you

**Pull request merged**